test/org/sagebionetworks/bridge/validators/ScheduleValidatorTest.java

```diff
@@ -102,4 +102,110 @@ public void activityCorrectlyParsesPublishedSurveyPath() {
         assertEquals("2015-01-27T17:46:31.237Z", ref.getCreatedOn());
     }
     
+    @Test
+    public void recurringSchedulesMustHaveCronTriggerOrInterval() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(ScheduleType.RECURRING);
+        try {
+            Validate.entityThrowingException(validator, schedule);
+        } catch(InvalidEntityException e) {
+            assertEquals("Schedule recurring schedules should have either a cron expression, or an interval", e.getErrors().get("Schedule").get(0));
+        }
+        
+        schedule.setInterval("P1D");
+        try {
+            Validate.entityThrowingException(validator, schedule);
+        } catch(InvalidEntityException e) {
```

Comment:
Should this throw or not? If it's supposed to throw, add fail() as mentioned above. If it's not supposed to throw, don't include the try catch.